catalog/spatial/registry/registry-identification-plugin/src/test/java/org/codice/ddf/registry/identification/IdentificationPluginTest.java

```diff
@@ -220,31 +225,102 @@ public void testDuplicateCheckingAfterDelete() throws Exception {
+        String xml = convert("/registry-both-extid.xml");
+        sampleData.setAttribute(Metacard.METADATA, xml);
+
+        OperationTransaction operationTransaction = new OperationTransactionImpl(null,
+                Collections.singletonList(sampleData));
+        Map<String, Serializable> properties = new HashMap<>();
+        properties.put(Constants.OPERATION_TRANSACTION_KEY, operationTransaction);
+        List<Map.Entry<Serializable, Metacard>> updatedEntries = new ArrayList<>();
+
+        MetacardImpl updateMetacard = sampleData;
+        updatedEntries.add(new AbstractMap.SimpleEntry<>(updateMetacard.getId(), updateMetacard));
+
+        UpdateRequestImpl updateRequest = new UpdateRequestImpl(updatedEntries,
+                Metacard.ID,
+                properties);
+        UpdateRequest processedUpdateRequest = idp.process(updateRequest);
+        assertThat(processedUpdateRequest.getUpdates()
+                .size(), is(0));
+    }
+
+    @Test
+    public void testSetTransientAttributesOnUpdateMetacard() throws Exception {
+        MetacardImpl previousMetacard = new MetacardImpl();
+        previousMetacard.setAttribute(Metacard.ID, "MetacardId");
+        previousMetacard.setAttribute(RegistryObjectMetacardType.REGISTRY_ID, "MetacardId");
+        previousMetacard.setAttribute(RegistryObjectMetacardType.PUBLISHED_LOCATIONS,
+                "Published Locations");
+        previousMetacard.setAttribute(RegistryObjectMetacardType.LAST_PUBLISHED,
+                "Last Published Time");
+        previousMetacard.setAttribute(Metacard.MODIFIED, new Date().from(Instant.now()));
+        OperationTransaction operationTransaction = new OperationTransactionImpl(null,
+                Collections.singletonList(previousMetacard));
+
+        Map<String, Serializable> properties = new HashMap<>();
+        properties.put(Constants.OPERATION_TRANSACTION_KEY, operationTransaction);
+        properties.put(RegistryConstants.TRANSIENT_ATTRIBUTE_UPDATE, true);
+        List<Map.Entry<Serializable, Metacard>> updatedEntries = new ArrayList<>();
+
+        MetacardImpl updateMetacard = new MetacardImpl();
+        updateMetacard.setAttribute(Metacard.ID, "MetacardId");
+        updateMetacard.setAttribute(RegistryObjectMetacardType.REGISTRY_ID, "MetacardId");
+        updateMetacard.setAttribute(Metacard.MODIFIED, new Date().from(Instant.now()));
+
+        updatedEntries.add(new AbstractMap.SimpleEntry<>(updateMetacard.getId(), updateMetacard));
+
+        UpdateRequestImpl updateRequest = new UpdateRequestImpl(updatedEntries,
+                Metacard.ID,
+                properties);
+        UpdateRequest processedUpdateRequest = idp.process(updateRequest);
+        Metacard processedMetacard = processedUpdateRequest.getUpdates()
+                .get(0)
+                .getValue();
+        assertThat(processedMetacard.getAttribute(RegistryObjectMetacardType.PUBLISHED_LOCATIONS)
```

Comment:
The update call to change a Registry Metacard doesn't actually occur in this plugin, this test is really just checking that a Registry Metacard that needs to be updated (with a newer modified date OR changed transient attributes) is not filtered out in the process call.

In this case both are changed and the metacard is still present in the update request.